plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -9,5 +9,7 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="lib" path="lib/httpclient-4.3.6.jar"/>
+	<classpathentry kind="lib" path="lib/httpcore-4.3.3.jar"/>
```

Comment:
if we switch to httpclient I guess we should at some point replace UrlConnectionClient to an implementation that's based on apache HttpClient to be consistent (reuse httpclient connection pooling etc.). Dare to file an issue in OSJC?